components/ResultsPage/index.tsx

```diff
@@ -35,18 +35,30 @@ const getEstimatedMonthlyTotalLinkText = (
   }
 }
 
-const getEligibility = (
+const getEligibilityText = (
   resultsArray: BenefitResult[],
   apiTsln: Translations,
   key: string
 ): string => {
+  return getEligibility(resultsArray, key)
+    ? `${apiTsln.benefit[key]}: ${apiTsln.result.eligible}`
+    : `${apiTsln.benefit[key]}: ${apiTsln.result.ineligible}`
+}
+
+const getEligibility = (
+  resultsArray: BenefitResult[],
+  key: string
+): boolean => {
   const eligibityResult = resultsArray.find((r) => r.benefitKey === key)
     .eligibility.result
-
-  return eligibityResult === ResultKey.ELIGIBLE ||
+  if (
+    eligibityResult === ResultKey.ELIGIBLE ||
     eligibityResult === ResultKey.INCOME_DEPENDENT
```

Comment:
Here, can just return "eligibityResult === ResultKey.ELIGIBLE ||
    eligibityResult === ResultKey.INCOME_DEPENDENT" to get a truthy or false value